consensus/consensus_leader_msg.go

```diff
@@ -21,8 +21,6 @@ func (consensus *Consensus) constructAnnounceMessage() []byte {
 	}
 	consensusMsg := message.GetConsensus()
 	consensus.populateMessageFields(consensusMsg)
-	// n byte of block header
-	consensusMsg.Payload = consensus.block // TODO: send only block header in the announce phase.
```

Comment:
Announce payload should still contain block header data. Otherwise, announce message is totally irrelevant to the new proposed block.

Reply:
block header added